scss/main.scss

```diff
@@ -9,6 +9,7 @@ $ex-light-lavender: #e9e3ed;
 $dark-grey: #606060;
 $orange: #F9B7AE;
 $hero-blue: #8cd8ed;
+$polaroid-border: #f7f3ed;
```

Comment:
just FYI: best practice is to name these vars by the color ($tan in this case) bc it's likely we'll be using the color more than once (this tan will be utilized as a bg color in the footer)